dbfilter_from_header/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
please use the new split readme [maintainer-tools](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)